cli/generate_operator_keys.go

```diff
@@ -15,6 +18,13 @@ var generateOperatorKeysCmd = &cobra.Command{
 	Use:   "generate-operator-keys",
 	Short: "generates ssv operator keys",
 	Run: func(cmd *cobra.Command, args []string) {
+		encrypted, _ := cmd.Flags().GetBool("encrypted")
```

Comment:
Why do we need `encrypted` flag if `password` flag is enough? However, if we had just `password`, it'd be not clear, maybe `encryption-password`?